### ISSUE TITLE
Changes Required in the website

### ISSUE THREAD

**Issue by kajal1801** [hack4bengal/hack4bengal.github.io#57]
Hey folks,
We will soon add Season 2.0 details to the website but before that, we need to make sure that all the previous season's details are removed and stored on another page.
You'll have to replace the following sections with TBD(To Be Disclosed):
- Winners (remove this section entirely)
- Hackathon Timeline (TBD)
- Sponsors (TBD) (Keep the Sponsor us button, modify it if possible)
- Prizes (TBD)
- Judges (TBD)
- Team (Remove the know more button, also clear the teams page for now)
- Community Partners (TBD)

Also, add all the removed data into a new Archive Page in the same order and add it to the Nav bar.

Happy Hacktober!

**Comment by kajal1801** [comment]
@harsitagarwalla187 yes you have to create a new page named "Archive" and add all the previous data there.

**Comment by harsitagarwalla187** [comment]
I wanna try it. Can you aasign it to me?

**Comment by harsitagarwalla187** [comment]
![2](https://user-images.githubusercontent.com/84774840/194141987-884cf1ad-bc83-430e-b38f-04dd61f9ffbe.png)
![1](https://user-images.githubusercontent.com/84774840/194142124-29275c7c-be3e-43bd-b0f2-3c41a721c40e.png)

**Comment by kajal1801** [comment]
> ![2](https://user-images.githubusercontent.com/84774840/194141987-884cf1ad-bc83-430e-b38f-04dd61f9ffbe.png) ![1](https://user-images.githubusercontent.com/84774840/194142124-29275c7c-be3e-43bd-b0f2-3c41a721c40e.png)

Can you be more creative with TBD?

**Comment by harsitagarwalla187** [comment]
![3](https://user-images.githubusercontent.com/84774840/194145992-5d172d5d-d347-4cda-a302-bf488fe52cf0.png)

**Comment by harsitagarwalla187** [comment]
what type of styling are you expecting?

**Comment by harsitagarwalla187** [comment]
https://user-images.githubusercontent.com/84774840/194150146-a742fbdf-6c9e-4f37-97bd-b2e029e752ec.mp4

Can do some styling in css

**Comment by harsitagarwalla187** [comment]
https://user-images.githubusercontent.com/84774840/194153202-30c3226a-d8e0-4763-9dce-591732c097e3.mp4
